### PR TITLE
Update api branch with master changes

### PR DESCRIPTION
**Pull request by crosbymichael**

### CODE REVIEW

**Comment by crosbymichael** [comment]
who added vet!!! ;)

agh

File: sample_configs/host-pid.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "cgroups": {
         "allowed_devices": [
             {
+<<<<<<< HEAD
```
Comment (mrunalp) [inline sample_configs/host-pid.json]: conflict markers in this file.

File: sample_configs/userns.json [hunk 2]
```diff
@@ -135,33 +135,60 @@
                 "type": 99
             },
             {
+<<<<<<< HEAD
```
Comment (mrunalp) [inline sample_configs/userns.json]: Here, too.

**Comment by mrunalp** [comment]
@LK4D4 added it :)

**Comment by crosbymichael** [comment]
I updated all the sample configs

**Comment by mrunalp** [comment]
cool, build failures?

**Comment by crosbymichael** [comment]
I'm working on fixing the vet errors

**Comment by crosbymichael** [comment]
Fixed it by removing vet

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**